src/components/common/header/index.astro

```diff
@@ -4,10 +4,55 @@
 import Container from "@/layouts/container/index.astro"
 
 ---
-<header class="bg-black h-12">
+<header class="bg-black h-20 fixed z-50 w-full items-center">
     <Container>
-        <section class="flex items-center h-full">
-            <p class="text-white">Header</p>
+        <section class="flex items-center justify-between cursor-pointer">
+            <div class="flex">
+                <div class="h-18 w-40 py-4">
+                    <img src="./public/logo/Lightfield-logo.png" alt="">
+                </div>
+                <div>
+                    <img src="./public/logo/Lightfield-logo.png" alt="" class="h-18 w-40 py-5 md:hidden" onclick="dropdown()">
+                    <ul class="md:flex md:bg-black md:space-x-6 md:mr-20 md:py-6 md:space-y-0 md:px-0 py-5 pl-8 pr-20 text-base space-y-4 bg-white right-0 fixed z-50" id="submenu">
```

Comment:
@Nandiniverma2024 why is bg-black inside md? Is it something specific to the screen resolution?

Reply:
For small screen sizes, we have a white background for menu options